scripts/install.sh

```diff
@@ -665,7 +665,7 @@ function uninstall() {
             # remove user
             if getent passwd "${SYSTEM_USER}" > /dev/null; then
                 userdel -r -f "${SYSTEM_USER}"
-                groupdel -f "${SYSTEM_USER}"
+                groupdel "${SYSTEM_USER}"
```

Comment:
The `-f` was there so the command doesn't error if the group doesn't exist. If we want to get rid of it, I think we should do:
```suggestion
                groupdel "${SYSTEM_USER}" 2>/dev/null || true
```

Also gets rid of the annoying error message.